openfisca_core/tests/test_reforms.py

```diff
@@ -101,8 +101,8 @@ def apply(self):
         )
     simulation = scenario.new_simulation(reference = True)
     reform_simulation = scenario.new_simulation()
-    assert str(simulation.calculate('birth')[0]) == '1980-01-01'
-    assert str(reform_simulation.calculate('birth')[0]) == '1970-01-01'
+    assert str(simulation.calculate('birth', None)[0]) == '1980-01-01'
+    assert str(reform_simulation.calculate('birth', None)[0]) == '1970-01-01'
```

Comment:
To carry on the implementation of the period, the cleanest way would be to define a new "eternity period" as a triplet(u"eternity", None, None). I confirm that this would be a lot of work, requiring to update everything in the core.